multiversx_sdk_cli/cli_contracts.py

```diff
@@ -232,6 +241,8 @@ def _add_arguments_arg(sub: Any):
     sub.add_argument("--arguments", nargs='+',
                      help="arguments for the contract transaction, as [number, bech32-address, ascii string, "
                      "boolean] or hex-encoded. E.g. --arguments 42 0x64 1000 0xabba str:TOK-a1c2ef true erd1[..]")
+    sub.add_argument("--arguments-file", type=str, help="a json file containing the arguments. ONLY if abi file is provided. "
+                     "E.g. { 'to': 'erd1...', 'amount': 10000000000 }")
```

Comment:
The example in `e.g.` is not completely precise (the wrapping array is missing). Maybe point towards mxpy's cookbook (official docs)?

Reply:
indeed, added the wrapping array. we don't have anything in the cookbook regarding this topic (as far as I know)